src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-speak"></span><span class="flc-prefsEditor-speak-label heading-text"></span></h2>
-
+<p class="flc-prefsEditor-speak-choice-label"></p>
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
         <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
             <span class="fl-prefsEditor-switch-inner"></span>
         </span>
-        <span class="flc-prefsEditor-speak-choice-label"></span>
```

Comment:
In the original code, the span is included in the label, so it could be read by an AT. However, in the change, there is nothing provided for a label at all.